dist/pom.xml

```diff
@@ -301,7 +301,7 @@
                         </configuration>
                     </execution>
                     <execution>
-                        <phase>generate-resources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
Is it safe to change the execution stage ?
Maybe no need to change it. Instead inserting your `dependency` plugin behind this one would be enough.
Maven will execute them in sequence.